manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/metadata.yaml

```diff
@@ -138,3 +138,5 @@ data:
   {{ end }}
   mysql_host: "mysql"
   mysql_port: "3306"
+  username: "root"
```

Comment:
Do you know why pipeline standalone deployment works?
https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/metadata/metadata-configmap.yaml#L8

Reply:
from @numerology lightweight uses secret to store username/pwd which makes sense. 
https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/metadata/metadata-mysql-secret.yaml

I'm ok either we merge this in and use secret in subsequent pr or change in this pr.